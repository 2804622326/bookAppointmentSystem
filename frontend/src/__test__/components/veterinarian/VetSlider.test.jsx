import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import VetSlider from '../../../components/veterinarian/VetSlider.jsx';

test('renders slider snapshot', () => {
  const { asFragment } = render(<VetSlider veterinarians={[]} />);
  expect(asFragment()).toMatchSnapshot();
});
