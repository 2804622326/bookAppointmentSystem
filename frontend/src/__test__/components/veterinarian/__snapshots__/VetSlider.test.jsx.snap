// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders slider snapshot 1`] = `
<DocumentFragment>
  <main>
    <div
      class="carousel slide"
    >
      <div
        class="carousel-indicators"
      />
      <div
        class="carousel-inner"
      />
      <a
        class="carousel-control-prev"
        href="#"
        role="button"
        tabindex="0"
      >
        <span
          aria-hidden="true"
          class="carousel-control-prev-icon"
        />
        <span
          class="visually-hidden"
        >
          Previous
        </span>
      </a>
      <a
        class="carousel-control-next"
        href="#"
        role="button"
        tabindex="0"
      >
        <span
          aria-hidden="true"
          class="carousel-control-next-icon"
        />
        <span
          class="visually-hidden"
        >
          Next
        </span>
      </a>
    </div>
  </main>
</DocumentFragment>
`;
